README.md

```diff
@@ -65,12 +64,6 @@
 

```

Comment:
[OPTIONAL BUT HIGHLY RECOMMENDED]

- [ ] It would be nice if you add some more meaningful features in your Future Feature section to make your readme interesting and professional 👍